ts/utils/navigation.ts

```diff
@@ -5,6 +5,9 @@ import { Platform } from "react-native";
 export const IO_INTERNAL_LINK_PROTOCOL = "ioit:";
 export const IO_INTERNAL_LINK_PREFIX = IO_INTERNAL_LINK_PROTOCOL + "//";
 
+export const IO_FIMS_LINK_PROTOCOL = "iosso:";
```

Comment:
```suggestion
export const IO_FIMS_LINK_PROTOCOL = "iofims:";
```

What you think about finally switch all "sso" to the more appropriate "fims"?

Reply:
Every document explicitly says that the prefix has to be `iosso`
[Design Review](https://pagopa.atlassian.net/wiki/spaces/FIM/pages/474447886/DR+Federated+Identity+Management+System#Definizione-delle-CTA)
[Technical integration guide](https://pagopa.atlassian.net/wiki/spaces/IC/pages/536936506/Guida+Tecnica+all+integrazione+con+il+protocollo+di+federazione+FIMS#2.-Creazione-di-una-CTA)